rust/src/adapter/otelstdout.rs

```diff
@@ -26,9 +26,9 @@ impl OtelTraceCtx {
 }
 
 impl OtelAdapterContainer {
-    pub async fn start<T: 'static>(&self, linker: &mut Linker<T>) -> Result<OtelTraceCtx> {
+    pub async fn start<T: 'static>(&self, linker: &mut Linker<T>, data: &Vec<u8>) -> Result<OtelTraceCtx> {
```

Comment:
```suggestion
    pub async fn start<T: 'static>(&self, linker: &mut Linker<T>, data: &[u8]) -> Result<OtelTraceCtx> {
```

(same suggestion as above)